.github/actions/build/action.yaml

```diff
@@ -0,0 +1,48 @@
+name: build
+description: Build the wiki
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+    - name: Allow modern Yarn
+      run: |
+        corepack enable
+      shell: bash
+    # Dependencies manually cached because `actions/setup-node@v3` does not support corepack
+    - name: Cache dependencies
+      uses: actions/cache@v3
+      with:
+        path: |
+          .yarn/cache
+        key: wiki-dependencies-${{ hashFiles('yarn.lock') }}
+        restore-keys: |
+          wiki-dependencies
+    - name: Install dependencies
+      run: |
+        yarn
+      shell: bash
+    - name: Build plugins
+      run: |
+        yarn build:plugins
+      shell: bash
+    - name: Build theme
+      run: |
+        yarn build:theme
+      shell: bash
+    # Artifacts cached after install to prevent removal by Yarn
+    - name: Cache build artifacts
+      uses: actions/cache@v3
+      with:
+        path: node_modules/.cache
+        key: wiki-build-artifacts-${{ hashFiles('package.json', 'yarn.lock', 'src/**/*.*') }}
+    # Create reference docs
+    #- name: Build reference docs
+    #  run: |
+    #    ./scripts/gen_${{ inputs.environment }}_references.sh
```

Comment:
Just to check, these were commented out for a reason right?

Reply:
Because I don't see the refs being built anywhere else.

Reply:
Good point. I commented them out as a quick fix, but I need to read them.